docs/source/tutorials/publication_quality_figures.rst

```diff
@@ -44,33 +44,25 @@ which completely resides on Python tools.
 
       ``anscombe.py``
 
-2. *Export to SVG*
+2. *Exporting to SVG*
 
-   A nice feature of matplotlib is that it allows to export figure to
+   A nice feature of matplotlib is that on can export figures to
    Scalable Vector Graphics (SVG) which is an open  vector format [1]_
    understood by many applications (such as Inkscape, Adobe
-   Illustrator or even web browsers). Not going too much into details,
-   I will only say that SVG files are text files with special
-   predefined tags (much alike HTML tags). You may try to open one of
-   them in a text editor to find out what I mean.
-
-3. *Arrange plots into composite figures*
-
-   Now, we would like to combine both plots into one figure and add
-   some annotations (such as one-letter labels: A,B, etc.). To this
-   end, I will use a small Python package I wrote with this purpose
-   ``svgutils``. It is written completely in Python and uses only
-   standard libraries. You may download it from `github
+   Illustrator or even web browsers). In a nutshell, SVG files are text files with special
+   predefined tags (similar to HTML tags). One can open it in a text editor too.
+
+3. *Arranging plots into composite figures*
+   
+   Using ``svgutils``, one can combine both plots into one figure and add
+   some annotations (such as one-letter labels: A,B, etc.). Download ``svgutils`` from `github
    <https://github.com/btel/svg_utils>`_.
 
-   The basic operations are similar to what you would do in a vector
-   graphics editor, but instead of using a mouse you will do some
-   scripting (I am sure you love it as much as I do). It may take some
-   more time at the beginning, but with
-   the advantage that you will not have to repeat the process when,
-   for some reason, you need to modify the plots you generated
+   The basic operations are similar to what one would do in a vector
+   graphics editor but using scripts instead of a mouse cursor. This reduces repitition as one will not have to repeat the process when,
+   for some reason, one needs to modify the plots they generated
    with matplotlib (to add more data or modify the
-   parameters of your analysis, just to name a few reasons).
+   parameters to the analysis, for example).
```

Comment:
i think "to the analysis" makes little sense, what is meant here is that we might want to modify the parameters that we used to generate the figures

Reply:
Would "parameters of the existing analysis" make sense?